presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-hdp3/apply-site-xml-override.sh

```diff
@@ -19,6 +19,6 @@ test -f "${site_xml}" || fail "${site_xml} does not exist or is not a file"
 test -f "${overrides}" || fail "${overrides} does not exist or is not a file"
 test ! -e "${site_xml_new}" || fail "${site_xml_new} already exists"
 
-xsltproc --param override-path "'${overrides}'" /docker/presto-product-tests/conf/docker/files/site-override.xslt "${site_xml}" > "${site_xml_new}"
+xsltproc --param override-path "'${overrides}'" site-override.xslt "${site_xml}" > "${site_xml_new}"
```

Comment:
If you reference `site-override.xslt` like this

- `/full/path/to/site-override.xslt` or
- `"${BASH_SOURCE%/*}/site-override.xslt"`

you don't need to `cd` in the `hadoop-entrypoint.sh`